Octokit.Tests/Clients/DeploymentStatusClientTests.cs

```diff
@@ -202,7 +209,7 @@ public void SendsPreviewAcceptHeaders()
 
             connection.Received(1).Post<DeploymentStatus>(Arg.Is<Uri>(u => u.ToString() == expectedUrl),
                                                           Arg.Any<NewDeploymentStatus>(),
-                                                          Arg.Is<string>(s => s == AcceptHeaders.DeploymentApiPreview));
+                                                          Arg.Is<string>(s => s == AcceptHeaders.Concat(AcceptHeaders.DeploymentApiPreview, AcceptHeaders.DeploymentStatusesPreview)));
```

Comment:
Can we use the string values here rather than the `AcceptHeaders.xxx` variables?  (I realise you were just updating what is already there, but we must have somehow missed this previously!)